src/main/java/teammates/storage/sqlentity/UsageStatistics.java

```diff
@@ -0,0 +1,138 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a system usage statistics for a specified period of time.
+ *
+ * <p>Note that "system usage" here is defined as user-facing usages, such as number of entities created
+ * and number of actions, as opposed to system resources such as hardware and network.
+ */
+@Entity
+@Table(name = "UsageStatistics")
+public class UsageStatistics extends BaseEntity {
+    @Id
+    private String id;
```

Comment:
This should be int with the generated tag, can refer to Course.

Reply:
I see that `id` is String in Course, is UUID in Notification, is Long in FeedbackSession. Why do we need to have different types of id?

Reply:
Ah, refer to feedback session instead. The default for all will be int (serial), unless there's a good reason for it. This is according to the db schema in initial design.